tests/suites/test_suite_aria.function

```diff
@@ -0,0 +1,318 @@
+/* BEGIN_HEADER */
+#include "mbedtls/aria.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_ARIA_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
+void aria_encrypt_ecb( char *hex_key_string, char *hex_src_string,
+                           char *hex_dst_string, int setkey_result )
```

Comment:
Should we align the arguments here?


```C
void aria_encrypt_ecb( char *hex_key_string, char *hex_src_string,
                       char *hex_dst_string, int setkey_result )
```

If so, there are many other places in this file that need alignment.

Reply:
Will fix that and other places.